.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,34 @@
+# MIT License
+#
+# Copyright (c) 2018 sclorg team at Red Hat
+#
+# Upload a Python package when a release is created
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows
+
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+
+      - name: Build a source tarball and a binary wheel
+        # https://pypa-build.readthedocs.io
+        run: |
+          python -m pip install build
+          python -m build --sdist --wheel
```

Comment:
Will this install all the `requirements.txt` entries?

Reply:
Not sure what is wrong, but I see this on my host:
```
$ python3 -m build --sdist --wheel
....
  File "/usr/lib64/python3.10/runpy.py", line 269, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/usr/lib64/python3.10/runpy.py", line 96, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/lib64/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "bin/dg", line 12, in <module>
    import six
ModuleNotFoundError: No module named 'six'
```

Reply:
Thanks for catching this...I will further investigate it.

Reply:
Can we just use `python3 setup.py sdist` instead, at least for now...?

Reply:
Yes, meanwhile I too came to this solution, because for python build we would (at least I think) need to have the `pyproject.toml`

Reply:
Adressed.